### PR TITLE
first shot fixing broken links

### PR DESCRIPTION
**Pull request by mmattel**
This PR is the first shot fixing broken links.

**It fixes:**
- wrong used ``link`` instead of ``xref``
- missing or wrong in page references for ``xref``
- missing or wrong out of page references for ``xref``
- correcting out of module references for ``xref``
- wrong or mistyped external target urls
- missing ``link:`` before http(s)
- some quick text fixes
- ...

**In a loop you do:**
```
antora generate --clean --pull --quiet --silent site.yml
linkchecker --no-status --complete --check-extern http://localhost:5000 > linkchecker.log
in directory /var/www/owncloud/docs
grep -r "link:" | grep -v "link:h" >> ../../xlink.log

step by step grep results from xlink.log against linkchecker.log
```

``grep`` finds all files containing the text ``link:`` and from the result exclude all matches ``link:h``
This finds all (possible) issues where there is:
``link`` used instead of ``xref`` (``link:http`` ... versus ``xref:installation/``...

**Manually edit the files**

There are "a million" broken links ``Edit this page``, see issue:https://github.com/owncloud/docs-ui/issues/9 (Edit this page...)

### CODE REVIEW

**Comment by settermjd** [comment]
I think there's something different about our builds. Here's why:

```console
Start checking at 2018-11-09 12:20:49+002

Statistics:
Downloaded: 7KB.
Content types: 0 image, 7 text, 0 video, 0 audio, 2 application, 0 mail and 5 other.
URL lengths: min=21, max=1519, avg=326.

That's it. 14 links in 5 URLs checked. 0 warnings found. 0 errors found.
Stopped checking at 2018-11-09 12:20:51+002 (2 seconds)
```

**Comment by mmattel** [comment]
have a look in your local repro in ``modules/administration_manual/pages/appliance/Clamav.adoc`` and compare to that what I have found/changed (it is the first file in the commit). The current link cant work and is fixed by my change.

**Comment by settermjd** [comment]
I can only assume that that works, as it's not an Xref, so the pattern will never be matched by the Xref validator, hence it not turning up in the invalid link results.

File: modules/administration_manual/pages/configuration/server/antivirus_configuration.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 [[overview]]
 == Overview
 
-http://www.clamav.net/index.html[ClamAV] is the only _officially_
+link:http://www.clamav.net/index.html[ClamAV] is the only _officially_
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/antivirus_configuration.adoc]: Afaik, external links don't need to be prefixed with `link:`.

File: modules/administration_manual/pages/configuration/user/user_auth_ftp_smb_imap.adoc [hunk 2]
```diff
@@ -23,9 +23,9 @@ Currently the https://github.com/owncloud/apps[External user support
 app] (user_external), _which is not enabled by default_, provides three
 backends. These are:
 
-* link:imap[IMAP]
-* link:smb[SMB]
-* link:ftp[FTP]
+* xref:imap[IMAP]
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/user/user_auth_ftp_smb_imap.adoc]: This don't appear valid, as xref's need to either end with `.adoc` or `#name-of-a-valid-anchor`.

**Pull request merged**